src/main/kotlin/day3/Cleric.kt

```
@@ -0,0 +1,42 @@
+import kotlin.math.max
+import kotlin.math.min
+import kotlin.random.Random
+
+class Cleric(
+    var name: String,
+    val maxHp: Int = 50,
+    var hp: Int = maxHp,
+    val maxMp: Int = 10,
+    var mp: Int = maxMp,
+) {
+    fun attack() {}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)